syntax = "proto3";

service MessageService {
    rpc AddMessage (Message) returns (MessageResponse);
}

message Message {
    int32 id = 1;
    string type = 2;
    string routing_key = 3;
    string content = 4; 
}

message MessageResponse {
    int32 status_code = 1; 
}